Wikipedia/Images.xcassets/share-building.imageset/Contents.json

```diff
@@ -2,18 +2,7 @@
   "images" : [
     {
       "idiom" : "universal",
-      "filename" : "shareIllo.png",
-      "scale" : "1x"
-    },
-    {
-      "idiom" : "universal",
-      "filename" : "shareIllo@2x.png",
-      "scale" : "2x"
-    },
-    {
-      "idiom" : "universal",
-      "filename" : "shareIllo@3x.png",
-      "scale" : "3x"
+      "filename" : "group4.pdf"
```

Comment:
May want a more descriptive file name here.

Reply:
These filenames aren't really relevant - they're obscured by the asset catalog.

Reply:
Ah, fair enough.